CMakeLists.txt

```diff
@@ -78,6 +78,26 @@ if (NOT DEFINED RTRLIB_TRANSPORT_SSH OR RTRLIB_TRANSPORT_SSH)
     endif(LIBSSH_FOUND)
 endif(NOT DEFINED RTRLIB_TRANSPORT_SSH OR RTRLIB_TRANSPORT_SSH)
 
+# bgpsec
+if(DEFINED WITH_BGPSEC AND NOT WITH_BGPSEC)
+    message(STATUS "building librtr without BGPsec support")
+else()
+    find_package(OpenSSL 1.0 QUIET)
+
+    if(OPENSSL_FOUND AND OPENSSL_CRYPTO_LIBRARY)
+        set(RTRLIB_BGPSEC_ENABLED 1)
+        set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -D RTRLIB_BGPSEC_ENABLED=${RTRLIB_BGPSEC}")
```

Comment:
Apparently, `rtr_mgr.h` does not include `rtrlib.h` and thus `RTRLIB_BGPSEC_ENABLED` is not present. It must be, though, because at the bottom of `rtr_mgr.h` the BGPsec functions are encapsulated by an #ifdef. Not sure how to resolve this...

Reply:
I see, maybe create a `config.h` and put the `#cmakedefine`s there?

Reply:
I made some changes in this regard (and unfortunately already squashed them):

Create a `config.h.cmake` which solely holds `RTRLIB_BGPSEC_ENABLED`. `config.h` is consequently included everywhere where the define is required, namely
- `rtrlib/rtr_mgr.h`
- `rtrlib/rtr/packets.c`
- `rtrlib/rtr_mgr.c`
- `rtrlib/rtrlib.h`
I'm not sure if this is exactly what you meant. Can't wait to hear from you.